### DISCUSSION TITLE
solderless case for openDTU

### DISCUSSION THREAD

**Discussion by raccoon0815** [tbnobody/OpenDTU#342] [Show and tell]
Hi, I have created a solderless 3d printed case for openDTU. So every part can be recycled again in the future. There is also a variant for the EBYTE E01-ML01DP5 radio module.

update: new url
[https://www.thingiverse.com/thing:5632374](https://www.thingiverse.com/thing:5632374)

have fun :-)

**Comment by darkl0rdchen** [comment]
cool case! my 3D printer is already working.
my 3D printer is building it right now :-)

Which radio module would you recommend?

**Reply by raccoon0815** [reply]
In my case, I have had the best experiences with the EBYTE module.
Subjectively, I would say that the module has better radio reception.
I will do more detailed tests over the weekend.

**Reply by gilladur** [reply]
Did you find out any more information with your testing?

**Reply by raccoon0815** [reply]
https://github.com/tbnobody/OpenDTU/discussions/414
